components/ReferenceManager/references/ReferencesContainer.tsx

```diff
@@ -22,6 +23,8 @@ export default function ReferencesContainer({}: Props): ReactElement {
     application: "REFERENCE_MANAGER",
     shouldRedirect: true,
   });
+  const router = useRouter();
+  const currentProjectName = router.query.project_name;
```

Comment:
I realize there are some instances in our codebase where we use snake case but the vest majority is combo of snake case and kebab case. Can we update for consistence? e.g. `proejctName` instead of `project_name` ?

Reply:
hmm `project_name` in this case is query param in the URL. Not sure if using kebab case is a common practice in the URL